tests/tools/foundationdb.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+readonly workspace="/tmp/saunafs-fdb-test"
+
+function create_workspace() {
+    mkdir -p "${workspace}"/{conf,data,logs}
+    chown -R $(whoami):$(whoami) "${workspace}"
+}
+
+function create_config_file() {
+  cat > "${workspace}/conf/foundationdb.conf" <<EOF
+[fdbmonitor]
+user = $(whoami)
+group = $(whoami)
+lockfile = ${workspace}/fdbmonitor.pid
+
+[general]
+restart-delay = 60
+cluster-file = ${workspace}/conf/fdb.cluster
+
+[fdbserver]
+command = /usr/sbin/fdbserver
+public_address = 127.0.0.1:\$ID
+listen_address = public
+datadir = ${workspace}/data/\$ID
+logdir = ${workspace}/logs
+
+[fdbserver.4500]
+[fdbserver.4501]
+
+[backup_agent]
+command = /usr/lib/foundationdb/backup_agent/backup_agent
+logdir = ${workspace}/logs
+
+[backup_agent.1]
+EOF
+}
+
+function check_cluster_status() {
+  fdbcli --exec "status" --cluster-file "${workspace}/conf/fdb.cluster"
+}
+
+function cleanup_fdb_cluster() {
+    sudo pkill -f fdbmonitor || true
+    rm -r "${workspace}" || true
```

Comment:
Please, add a safety measure on `rm` calls
```suggestion
    rm -r "${workspace:?}" || true
```

Reply:
Done